tutorials/tour/_posts/2017-02-13-unified-types.md

```diff
@@ -48,3 +48,15 @@ c
 true
 <function>
 ```
+
+## Nothing and Null
+`Nothing` is a subtype of all types. There is no value that has type `Nothing`. There are two uses: The first is to signal abnormal termination such as an exception (i.e. it doesn't return). The second type usage is an element type of empty collections (e.g. `Set[Nothing]`).
+
```

Comment:
I think `Set[Nothing]` is probably not a good example because it happens that (even immutable) `Set` is invariant in Scala. You could use `Nil` (which has type `List[Nothing]`), alternatively.

I’m not sure we should mention this use case of `Nothing` here, though… You can not really explain it without explaining how does variance work.